src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let internalSum = 0;
+
+  const adder = (number) => {
+    const localSum = internalSum;
```

Comment:
The variable `localSum` is unnecessary since its value is always equal to `internalSum` at the time of assignment. You can directly return `internalSum` and reset it instead of using `localSum`.